stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: nightly-2016-06-21
+resolver: lts-9.21
```

Comment:
I had to bump the resolver to a slightly newer version, because nightly-2016-06-21 uses ghc-8.0.1 but actions/setup-haskell only supports 8.0.2 but not 8.0.1. Should we take the opportunity to bump to a much more recent LTS?

Reply:
Yes!